### PR TITLE
Check in the submit-queue health graphing scripts.

### PR DESCRIPTION
**Pull request by rmmh**
It generates this image:
http://storage.googleapis.com/kubernetes-test-history/k8s-queue-health.png

### CODE REVIEW

File: jenkins/queue-health/graph.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
```
Comment (spxtr) [inline jenkins/queue-health/graph.py]: I think we want a space after this.

File: jenkins/queue-health/graph.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+# Copyright YEAR The Kubernetes Authors All rights reserved.
```
Comment (spxtr) [inline jenkins/queue-health/graph.py]: 2016

File: jenkins/queue-health/graph.py (hunk 1 above)
Comment (rmmh) [inline jenkins/queue-health/graph.py]: A newline between the shebang line and the copyright?

Comment (spxtr) [inline jenkins/queue-health/graph.py]: I think so, yes.

Comment (rmmh) [inline jenkins/queue-health/graph.py]: Done.

Comment (spxtr) [inline jenkins/queue-health/graph.py]: The shebang says python 3 but I think this script also works with python 2?

**Comment by spxtr** [comment]
I don't think this needs to go under `jenkins/`, and it's probably a good idea to add a simple `README.md` saying what it does, because it's not super clear from the code.

**Comment by rmmh** [comment]
Moved out of jenkins, added a simple README.

**Pull request merged**